src/Modules.py

```diff
@@ -422,7 +435,9 @@
                          'Adaptor2d', 'GeomAdaptor', 'Geom', 'TopLoc',
                          'Geom2d'],
             ['BRepMesh_WireInterferenceChecker', 'BRepMesh_EdgeTessellator',
-             'BRepMesh_EdgeTessellationExtractor', 'BRepMesh_EdgeParameterProvider' ],
+             'BRepMesh_EdgeTessellationExtractor', 'BRepMesh_EdgeParameterProvider',
+             'BRepMesh_DataStructureOfDelaun', 'BRepMesh_DiscretRoot',
+             'BRepMesh_FastDiscret', 'BRepMesh_FastDiscretFace', 'BRepMesh_IEdgeTool'],
```

Comment:
Is it necessary to remove additional classes ? ```BRepMesh_DiscretRoot``` is important, I suggest we don't include this change, if possible

Reply:
I actually don't know, why this was removed. Is started to work on this using pythonocc 0.17, maybe they were removed there. I will put them back in.